crates/cheatcodes/src/inspector.rs

```diff
@@ -1210,6 +1217,23 @@ where {
         }
     }
 
+    /// Whether the given slot of address with arbitrary storage should be overwritten.
+    /// True if address is marked as and overwrite and if no value was previously generated for
+    /// given slot.
+    pub fn overwrite_arbitrary_storage(&self, address: &Address, storage_slot: U256) -> bool {
```

Comment:
```suggestion
    pub fn should_overwrite_arbitrary_storage(&self, address: &Address, storage_slot: U256) -> bool {
```

Reply:
:+1: changed in https://github.com/foundry-rs/foundry/pull/10219/commits/7741ba5b25839b23e9f3cdf02a3344a099baa81f